dlna/dms/cds_unix.go

```diff
@@ -0,0 +1,25 @@
+//+build unix
+
+package dms
+
+import (
+	"strings"
+
+	"golang.org/x/sys/unix"
+)
+
+func isHiddenPath(path string) (bool, error) {
+	return strings.Contains(path, "/."), nil
+}
+
+func isReadablePath(path string) (bool, error) {
+	err := unix.Access(path, unix.R_OK)
```

Comment:
I was thinking about this, it is technically ideal, but there are caveats that it's not guaranteed to be correct. I'm keen to try it anyway.

Reply:
Where can I read these caveats ? Trying to guess the access by analyzing the permissions seems to be awfully complex.

Reply:
Just in the man pages